suclass/cms/templates/body-post.html

```diff
@@ -1 +1,12 @@
-<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=8ca24832-80a4-4e66-ba71-c44c36c96b85"></script>
+<script>
+	$(document).ready(
```

Comment:
Given how wonky/non-obvious this fix/solution is, let's give a little notice here via comments.

Something like:

> We have to wait to append the script to the page until a little while _after_ the page has fully loaded, else SOMETHING SOMETHING BLAH BLAH QUICK EXPLANATION OF RACE CONDITION BLAH BLAH